projects/plugins/jetpack/changelog/add-mailchimp-block-tests

```diff
@@ -0,0 +1,3 @@
+Significance: patch
+Type: compat
+Comment: Added block unit and parsing tests for map block
```

Comment:
Does it need a changelog entry? I think we've been specifying as such here if not. 🤔

Reply:
turned out on further reading of p1HpG7-bqa-p2 that for patch you can leave body empty as an indication no changelog is needed and add a 'Comment' instead to still outline reason for change.

Reply:
Nice, thanks for confirming!